ArduCopter/GCS_Mavlink.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <GCS_MAVLink/GCS.h>
+#include "Copter.h"
```

Comment:
It was all, all bad.

So I've made the method which was causing all the problems always compiled in (which it needed to be anyway because of Copter's just-disarm-the-vehicle code which triggers if you don't have `AP_ADVANCEDFAILSAFE_ENABLED` set.